rules/windows/defense_evasion_file_creation_mult_extension.toml

```diff
@@ -2,6 +2,7 @@
 creation_date = "2021/01/19"
 maturity = "production"
 updated_date = "2021/03/03"
+min_stack_version = "7.12.0"
```

Comment:
is this needed? Aren't we not tracking branches pre 7.13?

Reply:
I'm adding it to every change I make that opts-in to new stack functionality like `regex~`.

Also, I believe the backport logic checks for this anyway and we don't yet have a way of inspecting a query to know what stack version supports that version of EQL.

Reply:
my hope is that eventually we'll have this set on every rule in our repo

Reply:
maybe we should add a `min_stack_reason` to optionally keep track, since there can be multiple reasons, because you bring up a good point on how we should use this field for other compatibility limitations as well

Reply:
i hope that git blame helps out a bit with that